sklearn/tests/test_calibration.py

```diff
@@ -395,25 +395,21 @@ def test_sigmoid_calibration():
         _SigmoidCalibration().fit(np.vstack((exF, exF)), exY)
 
 
+# TODO(1.3): Remove warning filter.
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
You should not need this line anymore since we don't change the default value of `normalize`.

Reply:
Indeed, this was an oversight on my part, thank you for pointing this out! I have removed this unnecessary warning filter.